cmd/kube-version-change/version.go

```diff
@@ -28,6 +28,8 @@ import (
 	"runtime"
 
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest"
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1"
```

Comment:
To call the init() function in pkg/api/v1beta3/register.go, otherwise api.Scheme will not have v1 registered.